tests/test_files/yaml/working_pipelines/minimal_retriever.yml

```diff
@@ -1,79 +0,0 @@
-components:
```

Comment:
can you confirm that most of these removed files are not used in tests?
(My impression is that they were present for manual tests before proper tests were put in place)